inmysparetime.html

```diff
@@ -97,6 +123,35 @@ <h3 align='center'> Coding! </h1>
     <a href="https://twitter.com/beverleynewing" target="_blank">
         <img src="https://simplesharebuttons.com/images/somacro/twitter.png" alt="Twitter" />
     </a>
+
+<script>
```

Comment:
I would extract the script out to it's own file and require it via the `<script src=''>` tag then look at wiring up the events after the DOM has finished loading via the https://developer.mozilla.org/en/docs/Web/Events/DOMContentLoaded event.